### ISSUE TITLE
Add priors for biases

### ISSUE THREAD

**Issue by davharris** [davharris/mistnet#70]
Will be important if there are species that are never observed in the training set but we don't their occurrence probability in the test set to be zero.
